MortgageDueDate.java

```diff
@@ -0,0 +1,114 @@
+// Import scanner to Java
+import java.util.Scanner; 
```

Comment:
please rename the file using your initial, RA_MortgageDueDate.java, and move these 2 homework files from HW1 to correct folder, otherwise it will create merge conflicts

Reply:
Understood, xièxiè